app.js

```diff
@@ -19,6 +19,9 @@ app.use('/static', express.static('public'));
 
 app.get('/', (req, res) => res.render('index'));
 
+const dashboard = require('./routes/dashboard');
+app.get('/dashboard/:id/:page?', dashboard.view);
```

Comment:
Could you also add a unit test in https://github.com/googleinterns/step24-2020/blob/master/test/test-pages.js for verifing the routing and page content?